src/main/viz/PileupCoverageTrack.js

```diff
@@ -65,11 +65,21 @@ class CoverageTiledCanvas extends TiledCanvas {
       .nice();
   }
 
+  localYScaleForRef(ref: string): (y: number) => number {
```

Comment:
The padding created in the original yScaleForRef seems to cause the y-coordinate of the DataCanvasRenderingContext2D in the render function to be shifted off by 10 degrees. The original yScaleForRef is still used to generate the tick marks, so I create another one for rendering coverage tiles that shouldn't be used outside of context of coverage tiles.